jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/ContextHandler.java

```diff
@@ -2852,4 +2864,9 @@ public boolean handle(org.eclipse.jetty.server.Request coreRequest, Response res
             }
         }
     }
+
+    public Resource getResourceAsBaseName()
```

Comment:
How about calling this:
```suggestion
    public Resource getNestedResourceForeTempDirName()
```

Reply:
Done.